src/tests/utils/database-helpers.ts

```diff
@@ -0,0 +1,255 @@
+   */
+  public async insertUserSession(userSession: userSession): Promise<userSession> {
+    const refreshTokerExpiresAt = userSession.refreshTokenExpiresAt ?? `CURRENT_DATE + INTERVAL '1 day')`;
+
+    await this.orm.connection.query(`INSERT INTO public.user_sessions ("user_id", "refresh_token", "refresh_toker_expires_at") VALUES (${userSession.userId}, '${userSession.refreshToker}, '${refreshTokerExpiresAt}')`);
+
+    return userSession;
+  }
+
+  /**
+   * Inserts note settings mock to then db
+   *
+   * @param noteSettings - noteSettings object which contain all info about noteSettings (some info is optional
+   *
+   * If customHostname is not passed, custom_hostname will be set to null in the database
+   */
+  public async insertNoteSetting(noteSettings: noteSettings): Promise<noteSettings> {
+    const customHostname = noteSettings.customHostname ?? null;
+    const invitationHash = noteSettings.invitationHash ?? createInvitationHash();
+
+    noteSettings.invitationHash = invitationHash;
+
+    await this.orm.connection.query(`INSERT INTO public.note_settings ("note_id", "custom_hostname", "is_public", "invitation_hash") VALUES (${noteSettings.noteId}, '${customHostname}', ${noteSettings.isPublic}, '${invitationHash}')`);
+
+    return noteSettings;
+  }
+
+  /**
+   * Inserts note team mock to then db
+   *
+   * @param noteTeam - noteTam object which contain all info about noteTeam
+   */
+  public async insertNoteTeam(noteTeam: noteTeam): Promise<noteTeam> {
+    await this.orm.connection.query(`INSERT INTO public.note_teams ("user_id", "note_id", "role") VALUES (${noteTeam.userId}, ${noteTeam.noteId}, ${noteTeam.role})`);
+
+    return noteTeam;
+  }
+
+  /**
+   * Inserts note relation mock to then db
+   *
+   * @param noteRelation object which contain all info about noteRelation
+   */
+  public async insertNoteRelation(noteRelation: noteRelation): Promise<noteRelation> {
+    await this.orm.connection.query(`INSERT INTO public.note_relations ("note_id", "parent_id") VALUES (${noteRelation.noteId}, ${noteRelation.parentId})`);
+
+    return noteRelation;
+  }
+
+  /**
+   * Inserts editor tool mock to then db
+   *
+   * @param editorTool object which contain all info about editorTool (some info is optional)
+   *
+   * if no isDefault passed, then is_default would be false in database
+   */
+  public async insertEditorTool(editorTool: editorTool): Promise<editorTool> {
+    const isDefault = editorTool.isDefault ?? null;
+
+    await this.orm.connection.query(`INSERT INTO public.editor_tools ("name", "title", "export_name", "source", "is_default") VALUES ('${editorTool.name}', '${editorTool.title}', '${editorTool.exportName}', '${editorTool.exportName}', ${isDefault}')`);
+
+    return editorTool;
+  }
+
+  /**
+   * Truncates all tables and restarts all autoincrement sequences
+   */
+  public async truncateTables(): Promise<unknown> {
```

Comment:
Is it possible to change promise <unknown> to another?

Reply:
by default `orm.connection.query()` returns unknown, i think we don't need to return something special here